src/pageEditor/modListingPanel/ActionMenu.tsx

```diff
@@ -135,17 +137,21 @@ const ActionMenu: React.FC<ActionMenuProps> = ({
 
   return (
     <div className={styles.root}>
-      <SaveButton
-        ariaLabel={labelRoot ? `${labelRoot} - Save` : undefined}
-        onClick={onSave}
-        disabled={!isDirty || disabled}
-      />
-      <EllipsisMenu
-        ariaLabel={labelRoot ? `${labelRoot} - Ellipsis` : undefined}
-        items={menuItems}
-        classNames={{ menu: styles.menu, menuButton: styles.ellipsisMenu }}
-        portal={true}
-      />
+      {onSave && (
```

Comment:
NIT: we never want to embed `undefined` in jsx, which is what will happen if we don't wrap `onSave` in `Boolean()`
```suggestion
      {Boolean(onSave) && (
```

Reply:
This change causes a typing issue with the `onSave` prop. Do you recommend I suppress the error? Or how would you handle this?
![image](https://github.com/user-attachments/assets/0299dbe0-a708-4414-8820-3d7318fc1539)

Reply:
Use `!` on the prop value?

Reply:
Ah, figured out that `onSave != null` works and doesn't produce any lint warnings. That would still return `false` and not `undefined` in the JSX like you suggest 👍